learn/00_introduction.md

```diff
@@ -1,36 +1,55 @@
 # Introduction
 
-Week Goal: Get excited about the programme and what you are going to learnt over the next 12 weeks.
+## Day 1
+- Welcome
+  - Overview of curriculum
+  - Learning materials
+- Setup machines
+- Greenfield
+  - Confirm brief
+  - Confirm features/expectations with business
+  - Planning
+- Lunch
+- Pair programming intro
+  - Rules - pomo/breaks/driving/navigating
+  - Set expectation of 50% of time pairing
+- Work on greenfield
+- Deploy Greenfield to prod
 
-### Day 1:
-* Welcome to the Made Tech Academy
-* Get your laptop setup
-* Meet your academy colleagues
-* Overview of learning programme
-* Provided with reading material via Kindle (TBC what, example lists below)
-    - https://www.amazon.co.uk/gp/registry/wishlist/32U6H7XU7CQ36/ref=cm_wl_list_o_2?
-    - https://www.amazon.co.uk/gp/registry/wishlist/31W7SR5EK4Z6X/ref=cm_wl_list_o_3?
-    - https://www.amazon.co.uk/gp/registry/wishlist/3HO0RB9YA1IF/ref=cm_wl_list_o_0?
-* Company lunch and drinks to get to know wider team
+## Day 2
+- Business intro
+  - Intro to ongoing-dev
```

Comment:
@RoryMacDonald would that maybe be something we could cover during the welcome on day 1?